packages/hash/frontend/src/components/hooks/useAuthenticatedUser.ts

```diff
@@ -47,9 +49,14 @@ export const useAuthenticatedUser = (
 
   const authenticatedUser = useMemo<AuthenticatedUser | undefined>(
```

Comment:
I can't comment high enough but I _believe_ something may have gone wrong with the `loadingUser` variable of this. I've branched off this branch and it seems that `loadingUser` is always true on the `entity-editor.page.tsx` route

Reply:
Nevermind it seems like something else is going on.